Client/mods/deathmatch/logic/CClientWaterManager.cpp

```diff
@@ -104,6 +104,24 @@ bool CClientWaterManager::SetAllElementWaterLevel(float fLevel, void* pChangeSou
     return true;
 }
 
+bool CClientWaterManager::ResetAllElementWaterLevel()
+{
+    for (CClientWater* pWater : m_List)
+        pWater->ResetLevel();
+    return true;
+}
```

Comment:
Why is this function as a bool if it is unnecessary? The other functions are of type void and this one doesn't seem to need to return a value either